shared/logutil/logutil.go

```diff
@@ -0,0 +1,78 @@
+//Package logutil creates a logrus lib file logger instance that
+//write all logs that are written to stdout.
+package logutil
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	joonix "github.com/joonix/log"
+	"github.com/sirupsen/logrus"
+	prefixed "github.com/x-cray/logrus-prefixed-formatter"
+)
+
+var _ = logrus.Hook(&WriterHook{})
+
+// WriterHook is a hook that writes logs of specified LogLevels to specified Writer.
+type WriterHook struct {
+	LogLevels []logrus.Level
+}
+
+// Fire will be called when some logging function is called with current hook.
+// It will format log entry to string and write it to appropriate writer.
+func (hook *WriterHook) Fire(entry *logrus.Entry) error {
+	line, err := entry.String()
+	if err != nil {
+		return err
+	}
+	//simply call the file logger Println func after removing the new line char
```

Comment:
Remove this comment, it doesn't add any information

Reply:
I removed the comment. The linter on my machine complained about it. If the build job fails on the server due to this I'll have to a one liner.

Reply:
Yep, the linter made the build job fail. I had to add a one liner.